src/components/header/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Header.css';
+
+function Header() {
+  return (
+    <div className="sm:px-25 md:px-60 lg:px-60 header text-center h-screen place-items-center">
+      <h1 className="sm:text-3xl md:text-4xl lg:text-5xl">What will you make today?</h1>
+      <h2 className="sm:text-xs md:text-xl lg:text-2xl">
```

Comment:
- The <h2> text gets very small between a min-width of 640px and a max-width of 767px. The font size goes from 16px to 12px to 16px as you size down. For accessibility reasons it should be 16px or higher so it's easier to read on smaller screens.